great_expectations/core/usage_statistics/anonymizers/profiler_anonymizer.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from great_expectations.core.usage_statistics.anonymizers.anonymizer import Anonymizer
+from great_expectations.rule_based_profiler import RuleBasedProfiler
+
+
+class ProfilerAnonymizer(Anonymizer):
```

Comment:
@anthonyburdi anything missing from this implementation? Tried to mirror the other anonymizers